README.adoc

```diff
@@ -72,6 +72,9 @@ make test
 
 == Running
 
+[NOTE]
+Before running SWS, you must install and deploy link:https://istio.io[Istio] first. There are a few places that you can reference in order to learn how to do this such as link:https://github.com/redhat-developer-demos/istio-tutorial[here], link:https://blog.openshift.com/evaluate-istio-openshift/[here], and link:https://istio.io/docs/setup/kubernetes/quick-start.html[here]. There is also a link:hack/istio-install-openshift.sh[hack/istio-install-openshift.sh] script to help you install Istio if you are using OpenShift.
+
```

Comment:
I would place this note later in the readme.
I think the logical order should be: "the following section assumes the user has openshift installed" (the section below this) and only then "assumes the user has istio installed" and not vise versa.